README.md

```diff
@@ -26,6 +26,7 @@ The layer on master branch is compatible with the following Yocto version:
 
 Version  | Codename  | LTS
 -------- | --------- | ---
+5.0      | Scarthgap | Yes
 4.0      | Kirkstone | Yes
 3.4      | Honister  | No
 3.3      | Hardknott | No
```

Comment:
these two need to be removed from docs as you removed the layer compatibility with them

Reply:
done